setup.py

```diff
@@ -25,8 +25,8 @@
     version='0.1.0',
 
     description='Simple Python module for downloading fundamental financial data from financials.morningstar.com.',
-    long_description='Good Morning is intended to be used as an extension to QSToolKit (QSTK) library. By using QSTK you can easily download historical stock market data from Yahoo Finance. You can also download fundamental financial data from Compustat. However, most individuals and institutions do not have access to Compustat. Good Morning attempts to mitigate this limitation by providing a very simple Python interface for downloading fundamental financial data from financials.morningstar.com.',
-
+    long_description=long_description,
```

Comment:
No it's not.  That is a waaaay old commit.  looking at the date, this is around the time I was learning Python.  The setup.py file I pushed is totally different so I'm not understanding why this is here.  Weird.  Will try to figure out what's going on.  Sorry about that.